setup.py

```diff
@@ -394,11 +364,19 @@ def run(self):
             print("%s Style check passed" % self._result_marker)
 
 
+# Custom build commands
+#
+# These commands extend standard setuptools build procedure
+#
+sdc_build_commands = versioneer.get_cmdclass()
+sdc_build_commands['build_doc'] = SDCBuildDoc
 sdc_build_commands.update({'style': style})
+sdc_version = versioneer.get_version()
+sdc_release = 'Alpha (' + versioneer.get_version() + ')'
```

Comment:
You could use f-strings in such cases:
`'Alpha (' + versioneer.get_version() + ')'` -> `f'Alpha ({versioneer.get_version()})'` or `'Alpha ({})'.format(versioneer.get_version())`